modules/add_credit/src/Plugin/AddCreditEntityType/Developer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_m10n_add_credit\Plugin\AddCreditEntityType;
+
+use Drupal\apigee_edge\Entity\DeveloperInterface;
+use Drupal\apigee_m10n_add_credit\Annotation\AddCreditEntityType;
+use Drupal\apigee_m10n_add_credit\Plugin\AddCreditEntityTypeBase;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Defines a developer add credit entity type plugin.
+ *
+ * @AddCreditEntityType(
```

Comment:
I think a plugin is overkill for this use case. We have 2 known types here to deal with and won't have any more so we don't need discovery.

Reply:
I tried a non-plugin approach initially but there was a lot of module exists calls all over the place to handle teams, permissions, and the settable options for the field.